src/core/core.js

```diff
@@ -189,8 +189,12 @@ export default class Core {
         verticalKey: verticalKey
       });
     }
+    const locationRadius = this._getLocationRadius();
+    // The backend throws an error when locationRadius is 0
+    // Partially because of this, the SDK has decided to make locationRadius of 0
+    // mean "unset my locationRadius"
+    const nonZeroLocationRadius = locationRadius === 0 ? undefined : locationRadius;
```

Comment:
Having a `locationRadius` and a `nonZeroLocationRadius` is kind of confusing.

Reply:
ah yeah it is, nonZeroLocationRadius is gone now!